apps/auth_zero/tests.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from apps.common.config import ConfigFileLoader
+from apps.common.config import CredentialsLoader
```

Comment:
@codecakes One of the conflicts was whether to use `from apps.common.config import` or `from common.config import`. Having previously removed the `__init__.py` file from `common`, as it is now within the `apps` package, I opted to use the former. Do you agree with this?

Reply:
__init__.py has nothing do with whether a package is standalone or not if you define a folder as a module; if a path is a part of import, then all modules in that path should have __init__.py

Reply:
Okay. Thanks for the clarification. Am I right in thinking that I should still perform an absolute import even though I could import from `common`?

Reply:
Always perform absolute imports